modules/microshift-low-latency-config-yaml.adoc

```diff
@@ -0,0 +1,71 @@
+    numaNode: 0
+  kubeReserved:
+    memory: 500Mi
+  systemReserved:
+    memory: 500Mi
+  evictionHard: # <7>
+    imagefs.available: "15%" # <8>
+    memory.available: "100Mi" # <9>
+    nodefs.available: "10%" # <10>
+    nodefs.inodesFree: "5%" # <11>
+  evictionPressureTransitionPeriod: 0s
+# ...
+----
+<1> If you change the CPU or memory managers in the kubelet configuration, you must remove files that cache the previous configuration. Restart the host to remove them automatically, or manually remove the `- /var/lib/kubelet/cpu_manager_state` and `- /var/lib/kubelet/memory_manager_state` files.
+<2> The name of the policy to use. Valid values are `none` and `static`. Requires the `CPUManager` feature gate to be enabled. Default value is `none`.
+<3> A set of `key=value` for setting extra options that fine tune the behavior of the `CPUManager` policies.  Default: null. Requires both the `CPUManager` and `CPUManagerPolicyOptions` feature gates to be enabled.
+<4> The name of the policy used by Memory Manager. Default value is `none`. Requires the `MemoryManager` feature gate to be enabled.
+<5> Required. This parameter is essential to dividing the management and application workloads. Use this parameter to define a static CPU set for the host-level system and Kubernetes daemons, plus interrupts and timers. Then the rest of the CPUs on the system can be used exclusively for workloads. Use your TuneD profile to move the system and Kubernetes daemons, interrupts, and timers to the explicit CPU set defined by this option.
+<6> The value in `reservedMemory[0].limits.memory`, `1100` Mi in this example, is equal to `kubeReserved.memory` + `systemReserved.memory` + `evictionHard.memory.available`.
+<7> The `evictionHard` parameters define under which conditions the kubelet evicts pods. When you change the
+default value of only one parameter for the `evictionHard` stanza, the default values of other parameters are not inherited and are set to zero. Provide all the threshold values even when you want to change just one.
+<8> The `imagefs` is a filesystem that container runtimes use to store container images and container writable layers. In this example, the `evictionHard.imagefs.available` parameter means that the pod is evicted when the available space of the image filesystem is less than 15%.
+<9> In this example, the `evictionHard.memory.available` parameter means that the pod is evicted when the available memory of the node drops below 100MiB.
+<10> In this example, the `evictionHard.nodefs.available` parameter means that the pod is evicted when the main filesystem of the node has less than 10% available space.
+<11> In this example, the `evictionHard.nodefs.inodesFree` parameter means that the pod is evicted when more than 15% of the node's main filesystem's inodes are in use.
+
+.Verification
+
+. You can check that your settings are in the `config.yaml` file in the `/var/lib/microshift/resources/kubelet/config/` directory.
```

Comment:
Side note: this is after MicroShift starts. Not sure if we need to include it or it is obvious

Reply:
It is, but you never know.